qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -425,6 +722,10 @@ def add_parameter_value(
 
         self._params[ParameterKey(param_name, qubits, sched_name)].append(value)
 
+        if update_inst_map and schedule is not None:
+            param_obj = self.calibration_parameter(param_name, qubits, sched_name)
+            self._parameter_inst_map_update(param_obj)
```

Comment:
Should `_parameter_inst_map_update` allow for restriction by `qubits`? The parameter links are restricted to a `qubits` tuple, so it shouldn't be necessary to update the inst map for all the other `qubit` tuples.

Reply:
Hmmm it should be possible to do that but when I change `self.update_inst_map(schedules)` to `self.update_inst_map(schedules, qubits=qubits)` I start getting a few failures in the tests. I'm not sure why. I opened a bug report here: #558 . To make this change possible in a future PR I made a few minor changes here: f1bcc4f . Fixing #558 is a bit out of scope of this PR which is intended to lay the ground work for Calibration serialization. I would suggest fixing #558 in a dedicated PR to a) keep this PR small and b) make sure we do not bury the changes from the fix in those from this refactor.

Reply:
Sounds good.